package/boot/uboot-at91/Makefile

```diff
@@ -174,7 +174,6 @@ UBOOT_TARGETS := \
 define Build/Compile
 	+$(MAKE) $(PKG_JOBS) -C $(PKG_BUILD_DIR) \
 		CROSS_COMPILE=$(TARGET_CROSS) \
-		DTC=$(PKG_BUILD_DIR)/scripts/dtc/dtc \
```

Comment:
Because of U-Boot commit 93b196532254 ("Makefile: Only build dtc if needed") now DTC is not compiled by U-Boot if it is provided though DTC variable. From U-Boot tree:

commit 93b196532254366f653b4d763f69e49ff193f06c
Author: Simon Glass <sjg@chromium.org>
Date:   Wed Sep 22 11:34:44 2021 -0600

    Makefile: Only build dtc if needed
    
    At present U-Boot always builds dtc if CONFIG_OF_CONTROL is defined, even
    when DTC is provided. The built dtc is not actually used, so this is a
    waste of time.
    
    Update the Makefile logic to build dtc only if one is not provided to the
    build with the DTC variable. Add documentation to explain this.
    
    This saves about 3.5 seconds of elapsed time on a clean build of
    sandbox_spl for me.
    
    Signed-off-by: Simon Glass <sjg@chromium.org>

Reply:
If you inspect that referenced commit closely, then you'll notice, that they're now checking for `import libfdt` and this is probably going to bomb out on our buildbots as this Python3 library is not installed, I've hit [that issue as well](https://git.openwrt.org/?p=openwrt/staging/ynezz.git;a=blob;f=package/boot/uboot-sunxi/patches/300-force-pylibfdt-build.patch;h=2ee08f08f28bb64fb35b1e54d29e3c5e15c857e3;hb=d34ef1602f60281fbd0884eb15d9e7fdfcda09da).

Reply:
In other words, this should be build tested before merging.